code/main/server_main.cpp

```diff
@@ -15,14 +15,14 @@
 
 #include "server_tflite.h"
 
-//#define DEBUG_DETAIL_ON      
-
+#include "basic_auth.h"
 
+//#define DEBUG_DETAIL_ON      
```

Comment:
You could probably add this line without changing so much whitespace and comments?